ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -5454,6 +5454,47 @@ RDC:
   implements:
   - MONITORING
 
+CHWZOUC:
+  description: "Chilled water valve control with occupied/unoccupied setpoints and temperature monitoring."
+  is_abstract: true
+  opt_uses:
+  - chilled_supply_water_temperature_sensor
+  uses:  
+  - chilled_water_valve_percentage_command
+  - zone_air_temperature_sensor
+  - zone_air_occupied_temperature_setpoint
+  - zone_air_unoccupied_cooling_temperature_setpoint
+  implements:
+  - CONTROL
+  
+PERHWZOUC:
+  description: "Perimeter heating water valve control with occupied/unoccupied setpoints and temperature monitoring."
+  is_abstract: true
+  opt_uses:
+  - perimeter_heating_supply_water_temperature_sensor
+  - perimeter_heating_water_valve_percentage_sensor
+  uses:  
+  - perimeter_heating_water_valve_percentage_command
+  - zone_air_temperature_sensor
+  - zone_air_occupied_temperature_setpoint
+  - zone_air_unoccupied_heating_temperature_setpoint
+  implements:
+  - CONTROL
+  
+ZTOUC:
+  description: "Zone air temperature unoccupied/occupied control."
+  is_abstract: true
+  opt_uses:
+  - zone_air_relative_humidity_sensor
+  - zone_occupancy_status
+  uses:  
+  - zone_air_temperature_sensor
+  - zone_air_occupied_temperature_setpoint
```

Comment:
its odd to me that you have differentiated heating and cooling setpoints for unoccupied times, but there are no equivalent differentiations for occupied setpoints. 

in general, we see either a single setpoint that changes based on mode, or we see two effective setpoints (for cooling and for heating) that change based on occ/unocc status. what is the context for this weird configuration?

Reply:
There were no heating/cooling setpoints for occupied mode. Unit runs in either cooling or heating mode while zone is occupied and controls to the occupied setpoint. In unoccupied mode the heating/cooling valves modulate to maintain the zone between the unoccupied heating/cooling setpoints.

Reply:
I see this pretty frequently in terminal units where there is a single setpoint during occupied hours, but during unoccupied hours there is a night setback for both heating and cooling. Do they need to be separated into separate abstract types?